### ISSUE TITLE
Support for eightpoints/guzzle-bundle 8.x

### ISSUE THREAD

**Issue by petski** [gregurco/GuzzleBundleCachePlugin#12]
This bundle requires eightpoints/guzzle-bundle ~7.0. As of Jan 20th 2020 version 8.0 of eightpoints/guzzle-bundle came out.

Full error from composer when trying to bump to eightpoints/guzzle-bundle 8.x:

`gregurco/guzzle-bundle-cache-plugin v1.0.4 requires eightpoints/guzzle-bundle ~7.0 -> satisfiable by eightpoints/guzzle-bundle[7.3.0, v7.0.0, v7.1.0, v7.1.1, v7.2.0, v7.2.1, v7.3.1, v7.4.0, v7.4.1, v7.5.0, v7.6.0, v7.6.1, v7.x-dev] but these conflict with your requirements or minimum-stability.`

**Comment by petski** [comment]
While browsing the forks of this project I noticed that user @lmeyer created a fix already (in https://github.com/lmeyer/GuzzleBundleCachePlugin/commits/master). For @gregurco or those who are eager to give 8.x a try:

In composer.json

```json
    ...
    "repositories": [
        {
            "type": "vcs",
            "url": "git@github.com:lmeyer/GuzzleBundleCachePlugin"
        }
    ],
    ...
```

In shell:
```shell
composer require 'gregurco/guzzle-bundle-cache-plugin=dev-master#5943288'
```

**Comment by gregurco** [comment]
Hello @petski 
I created and merged new PR (#13) with proper changes. Also released them: https://github.com/gregurco/GuzzleBundleCachePlugin/releases/tag/v2.0.0
